examples/v1alpha3/metrics-with-timestamp/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM pytorch/pytorch:1.0-cuda10.0-cudnn7-runtime
+
+WORKDIR /var
```

Comment:
I don't believe /var is appropriate location for the script. If you check FSH (https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard) `/var` should be used for changing files.

I would suggest using `/opt` and ideally a subdir in there - e.g. `/opt/mnist/`

WDYT?

Reply:
Good catch. What do you think about `/app`?

Reply:
Any particular reason you do not want to go with existing directory structure in `/`? I don't think it is a good practice in linux systems to put code in "random" dirs in `/`, but feel free to convince me otherwise:)

Reply:
Do you mean in `/examples/v1alpha3/metrics-with-timestamp` ?

Reply:
> Do you mean in `/examples/v1alpha3/metrics-with-timestamp` ?

Is this answer to @johnugeorge'S question?:-)

Reply:
> Any particular reason you do not want to go with existing directory structure in `/`? I don't think it is a good practice in linux systems to put code in "random" dirs in `/`, but feel free to convince me otherwise:)

To this question :) 
Which `WORKDIR` you think will be better?

Reply:
Ah, sorry:) as I suggested in my first comment - I would go for `/opt/mnist/mnist.py` or to make it extremely obvious what it is (although that is already captured in the image name, but you never know..) `/opt/metrics-with-timestamp/mnist.py` or something along those lines

Reply:
@johnugeorge @hougangliu What do you think about `WORKDIR` for the example as `/opt/metrics-with-timestamp/mnist.py`